rakelib/artifacts.rake

```diff
@@ -95,6 +95,9 @@ namespace "artifact" do
     @exclude_paths << 'vendor/**/gems/*/test/**/*'
     @exclude_paths << 'vendor/**/gems/*/spec/**/*'
 
+    # dependency vulnerability scanners may pick unnecessary Gemfile.lock files
+    @exclude_paths << 'vendor/**/gems/*/Gemfile.lock'
```

Comment:
Let's also exclude 'Gemfile', as they're ignored (more on this here: https://bundler.io/guides/faq.html#using-gemfiles-inside-gems)

```suggestion
    @exclude_paths << 'vendor/**/gems/*/Gemfile.lock'
    @exclude_paths << 'vendor/**/gems/*/Gemfile'
```

Reply:
Good suggestion, thanks.
It seems we should also exclude the shared gems (ex: `vendor/jruby/lib/ruby/gems/shared/gems/net-telnet-0.1.1/Gemfile`) so the patter would be:
```
    @exclude_paths << 'vendor/**/gems/**/Gemfile.lock'
    @exclude_paths << 'vendor/**/gems/**/Gemfile'
```